Competition2017/src/competition/subsystems/drive/commands/DriveToOrientationCommand.java

```diff
@@ -0,0 +1,60 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.ContiguousHeading;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToOrientationCommand extends BaseCommand{
```

Comment:
I think this would be clearer if it were called `RotateToOrientationCommand` or `RotateToHeadingCommand`.